ui/src/app/navigation/commoncontrols.component.ts

```diff
@@ -13,11 +14,17 @@ export class CommoncontrolsComponent extends AbstractSearchComponent implements
   searchIcon = SvgHelper.path(SvgIcon.SEARCH)
   dismissIcon = SvgHelper.path(SvgIcon.DISMISS)
 
-  constructor(protected searchService: SearchService, protected route: ActivatedRoute) {
-    super(searchService, route)
+  constructor(protected searchService: SearchService, protected route: ActivatedRoute, protected authService: AuthService) {
+    super(searchService, route, authService)
   }
 
   ngOnInit(): void {
   }
 
+  // isDisabled(path : string): boolean {
```

Comment:
Can we please either put the commented code into service, or remove it?

Reply:
Removed.